<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kargoHesapla</class>
 <widget class="QWidget" name="kargoHesapla">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>422</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>547</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kargo Hesapla</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
 	 background: #3498db;
	border-radius: 18px;
	font-family: Courier New;
	color: #ffffff;
	font-size: 15px;
	padding: 10px 20px 10px 20px;
	text-decoration: none;
    max-width:120px;
	min-width:120px;
	max-height:20px;	 
	min-height:20px;	 
}

QPushButton:hover {
	background: #3cb0fd;
	text-decoration: none;
}
QLineEdit{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QComboBox{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QWidget{
		font-family: Courier New;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnHacim">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Hesapla</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGeri">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Geri</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblHacim">
         <property name="styleSheet">
          <string notr="true">font:bold 18px </string>
         </property>
         <property name="text">
          <string>Hacim(cm^3) :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnHacim">
         <property name="minimumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-radius:10px;
font:16px </string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblAgirlik">
         <property name="styleSheet">
          <string notr="true">font:bold 18px </string>
         </property>
         <property name="text">
          <string>Ağırlık(kg) :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnAgirlik">
         <property name="minimumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-radius:10px;
font:16px </string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font:bold 18px </string>
         </property>
         <property name="text">
          <string>Birim Fiyat :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnFiyatBirim">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-radius:10px;
font:16px </string>
         </property>
         <property name="text">
          <string>1.25</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tl">
         <property name="styleSheet">
          <string notr="true">font:bold 18px </string>
         </property>
         <property name="text">
          <string>TL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lblUcret">
         <property name="styleSheet">
          <string notr="true">font:bold 18px </string>
         </property>
         <property name="text">
          <string>Ücret:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lnUcret">
         <property name="minimumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>371</width>
           <height>31</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
